fundingmanager.go

```diff
@@ -1310,6 +1343,20 @@ func (f *fundingManager) waitForFundingConfirmation(completeChan *channeldb.Open
 		return
 	}
 
+	// Send confirmation details to confirmChan for use in announceChannelAfterConfirmations
+	confirmChan <- confDetails
+}
+
+// announceChannelAfterConfirmations marks a fully confirmed channel as open and
+// available for use in the database.  It also adds the channel to the fundingManager's
+// internal persistent state machine that tracks the remaining process of the channel
+// opening.  This is useful to resume the opening process in case of restarts.
+// Finally, the fundingLocked message is sent to the peer and the channel is announced
+// to the greater network.  This method should be called only after waitForFundingConfirmation
+// completes successfully.
+func (f *fundingManager) announceChannelAfterConfirmations(
```

Comment:
Before announcing the  channel, this method marks the channel opened in the db, and sends the `fundingLocked` message to the peer, so I think the name is a bit misleading. Maybe something like `processConfirmedFunding` could be more telling?

Reply:
I agree, I actually had trouble coming up with a name for this...